query/src/main/scala/filodb/query/exec/rangefn/AggrOverTimeFunctions.scala

```diff
@@ -830,3 +830,95 @@ class HoltWintersChunkedFunctionL(funcParams: Seq[FuncArgs]) extends HoltWinters
     }
   }
 }
+
+abstract class PredictLinearChunkedFunction(funcParams: Seq[Any],
```

Comment:
Would be great add some docs here.... links or descriptions of what predict linear is, and how the algorithm works

Reply:
https://github.com/filodb/FiloDB/pull/568/commits/7e03132cf261835bc5f8bc640a0f5860b9f38860